lib.ts

```diff
@@ -124,78 +138,95 @@ export type JobOptions = {
  */
 export class Job {
   /**
-   * Do not used. The _uuid property is unstable and should be considered an internal implementation detail.
-   * @internal
+   * @deprecated Do not use. The _uuid property is unstable and should be considered an internal implementation detail.
```

Comment:
Used [`@deprecated`](https://jsdoc.app/tags-deprecated.html) here so it's crossed out in autocomplete.
![image](https://user-images.githubusercontent.com/20016103/230681614-4280a102-3d24-4366-9654-8fce1d06af99.png)